apps/ledger-live-mobile/src/families/evm/EvmCustomFees/utils.tsx

```diff
@@ -0,0 +1,51 @@
+import BigNumber from "bignumber.js";
+import { getEnv } from "@ledgerhq/live-env";
+import { inferDynamicRange, Range } from "@ledgerhq/live-common/range";
+import { GasOptions } from "@ledgerhq/coin-evm/lib/types";
+import invariant from "invariant";
+
+// This is 10 Gwei in wei. 20 Gwei is 2e8 or 20e9
+// Should we rename the variable to TEN_GWEI or update the value to 20e9?
+const TWENTY_GWEI = new BigNumber(10e9);
```

Comment:
This is 10 Gwei in wei. 20 Gwei is `2e8` or `20e9`
Should we rename the variable to TEN_GWEI or update the value to 20e9?

Reply:
I lean more for the TEN_GWEI name, and then use 2*TEN_GWEI if needed